doc/tutorials/basic/join_community.md

```diff
@@ -0,0 +1,77 @@
+# Join the Community
+
+Find inspiration in the community forums, ask for help, help others, share your work, report issues, add feature requests, and contribute improvements.
+
+## Prerequisites
+
+We will assume you already have a [Github](https://github.com/) account. If not, please follow the official [Creating an account on Github](https://docs.github.com/en/get-started/quickstart/creating-an-account-on-github) guide.
```

Comment:
In theory you might be able to do without but in practice it would make it difficult to be a part of the community.

- You cannot report issues or feature requests.
- Signing up for discourse and other developer services are harder as you can't just log in using your Github account. The same goes for login.

The purpose of the *Join the community* is to nudge users to become members of the community. I believe a part of that is having a Github account.

I'm open for removing the comment if someone feel strongly that this should be removed.

Reply:
You could instead write a note that says that having a Github account helps creating accounts on all the platforms where the community chats.

It happened quite often I tried to look up someone's name on Discourse to find them on Github and failed, I don't think most people there either have a Github account or use it to create an account.

Reply:
I've removed the ask to create a github account